src/main/java/seedu/moneygowhere/logic/commands/FindCommand.java

```diff
@@ -16,19 +20,29 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all spending entries whose names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+            + "Not all fields are necessary, but they are combined together to form a query.\n"
+            + "Parameters: n/KEYWORD [MORE_KEYWORDS]...  d/DATE_RANGE c/COST_RANGE r/REMARK t/TAG\n"
+            + "Example: " + COMMAND_WORD + " n/chicken rice d/1/1/2019 to 2/1/2019 c/1.50-5 r/tasty t/food";
```

Comment:
Might be better to standardize cost range and date range format. Currently, one uses "to" and the other is "-".

Reply:
The date range format also allows "-" but there is a mandatory space required between the two dates, if not the input is unrecognised.